demos/build_demos_msvc.bat

```diff
@@ -122,8 +131,16 @@ cd "%SOLUTION_DIR64%" && cmake -G "Visual Studio !VS_VERSION!" -A %PLATFORM% %EX
 echo.
 echo ###############^|^| Build Open Model Zoo Demos using MS Visual Studio ^|^|###############
 echo.
-echo cmake --build . --config Release
-cmake --build . --config Release
+echo cmake --build . --config Release %BUILD_TARGETS%
```

Comment:
This does not correspond to the actual command(s) being run. You should remove this and add an echo command before each of the `cmake` commands below.